packages/ethereum-storage/src/gas-price-definer.ts

```diff
@@ -65,31 +64,52 @@ export default class GasPriceDefiner {
         ]);
       }
     }
-
+    else if (
+      networkName ===
+      EthereumUtils.getEthereumNetworkNameFromId(StorageTypes.EthereumNetwork.XDAI)
+      ) { 
+      const gasPrice: typeof bigNumber = await this.pollProviders(type, networkName);
+      if (gasPrice.length > 0) {
+        return gasPrice
+      } else {
+        this.logger.warn('Not able to parse the gas fees correctly', [
```

Comment:
"Not able to" looks weird for error message. @AdamDowson what do you think ?

Reply:
`Unable to parse gas fees` should suffice.

Reply:
thanks , noted